src/5.0_to_5.1/migrateFilters.ts

```diff
@@ -1,13 +1,38 @@
-import type { Mdx } from "@activeviam/activeui-sdk-5.0";
+import {
+  getTargetCube,
+  createFilter,
+  DataModel,
+} from "@activeviam/activeui-sdk-5.1";
+import { Mdx } from "@activeviam/activeui-sdk-5.0";
+import { getAllCubes } from "./getAllCubes";
+import { createFilterWithFirstMatchingCube } from "./createFilterWithFirstMatchingCube";
 
 /**
- * Wraps each filter in `filters` under an `mdx` property in an object.
- * Indeed, in 5.1 filters can have an extra `isPinned` property.
+ * In 5.0, a filter in a widget/dashboard state is just an {@link MdxExpression}.
```

Comment:
Then eslint will complain that it's imported but never used.
And if we cannot make it poka-yoke we're sure that this will not be respected in a bunch of places, so I prefer to keep it simple for now.

Reply:
Are you sure? I just quickly checked with a dummy file and it seems fine:
![image](https://user-images.githubusercontent.com/15254842/214909850-5df970ff-501e-453c-afe4-ae0d7ac63751.png)
![image](https://user-images.githubusercontent.com/15254842/214909913-c1c93604-5853-4e31-b4a0-9607349c13da.png)